lapce-data/src/proxy.rs

```diff
@@ -914,30 +914,6 @@ impl Remote for WslRemote {
     }
 }
 
-// Rust-analyzer returns paths in the form of "file:///<drive>:/...", which gets parsed into URL
-// as "/<drive>://" which is then interpreted by PathBuf::new() as a UNIX-like path from root.
-// This function strips the additional / from the beginning, if the first segment is a drive letter.
-#[cfg(windows)]
-pub fn path_from_url(url: &Url) -> PathBuf {
```

Comment:
I think this is for when you connect to a Linux remote from a Windows machine. The proper way to fix it is probably using Url throughout Lapce.